src/lightning/data/streaming/cache.py

```diff
@@ -19,7 +19,7 @@
 from lightning.data.datasets.env import _DistributedEnv
 from lightning.data.streaming.constants import (
     _INDEX_FILENAME,
-    _LIGHTNING_CLOUD_GREATER_EQUAL_0_5_50,
+    _LIGHTNING_CLOUD_GREATER_EQUAL_0_5_51,
```

Comment:
```suggestion
    _LIGHTNING_CLOUD_LATEST,
```
as we update with each release there is probably no need to have in version in var name

Reply:
This is mostly for traceability for now. We will have to stop doing this quite soon.